package.json

```diff
@@ -92,6 +92,7 @@
     "@types/autosize": "^4.0.1",
     "@types/blueimp-md5": "^2.18.0",
     "@types/clean-css": "^4.2.2",
+    "@types/gtag.js": "^0.0.12",
```

Comment:
We will need a minor version bump for this

Reply:
oh, ok

Reply:
Just confirm, I think you're talking about 

> \# * adding a dependency only requires a minor version bump;

A comment in `version.py`?

Reply:
Just did, may you please check again. Thank you Zixuan!!